packages/gasket-plugin-https/lib/configure.js

```diff
@@ -0,0 +1,22 @@
+/** @type {import('@gasket/core').HookHandler<'configure'>} */
+module.exports = function configure(gasket, config) {
+  const { root } = config;
+
+  function setRoot(serverConfig) {
+    if (!serverConfig) return;
+    if (serverConfig.root) return;
+
+    if (Array.isArray(serverConfig)) {
+      serverConfig.forEach(item => {
+        item.root = root;
```

Comment:
In the array, you should probably only conditionally set `root` since you haven't checked for it
```suggestion
        item.root ??= root;
```